package.json

```diff
@@ -1,17 +1,23 @@
 {
   "name": "@tram-one/tram-logo",
-  "version": "4.0.0",
+  "version": "4.1.0",
   "description": "This logo was created for the Tram-One Project, designed by Daniel Jurman",
   "main": "logo.svg",
   "files": [
     "dist",
     "variants",
     "logo.svg"
   ],
+  "source": "src/element.js",
+  "module": "dist/element.js",
+  "targets": {
+    "main": false
+  },
```

Comment:
these keys are used by `parcel` - specifically we are using `src/element.js` as the input, `dist/element.js` as the module export, and we should ignore the `main` key as a potential target (which parcel likes to default for the CommonJS export).

Reply:
This does mean that we aren't exposing a CommonJS export, but I'm not totally sure that's something that we have to worry about. I think those projects would have a hard time working with the way this project is formatted anyways, and if there is a request for that with a specific use-case, I'd be happy to make a follow up PR to handle that.